tests/bats-assert/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: bash
```

Comment:
I'm confussed. Is the stuff in `tests/*` not supposed to be here? The submodule is added to `test/bats` and IINM is the same as the files. If so, I suppose that `tests/*` should be deleted. Is that right?

Reply:
@travisspencer git submodule file got messed up since I changed the directory name and I added all the submodule files to the commit mistakenly. I have corrected it now.